layouts/_default/list.html

```diff
@@ -42,7 +42,15 @@ <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
                                           {{ end }}
                                           {{ if isset .Params "categories" }}
                                           {{ if gt (len .Params.categories) 0 }}
-                                          in <a href="{{ "categories/" | relURL }}{{ index .Params.categories 0 | urlize | lower }}">{{ index .Params.categories 0 }}</a>
+                                          in 
```

Comment:
I would like to not repeat code, could you replace the two ranges into one?

Like this example: https://stackoverflow.com/questions/21305865/golang-separating-items-with-comma-in-template

Reply:
Thanks for pointing at that. changed.